webui.py

```diff
@@ -680,6 +681,11 @@ def flag(self, flag_data, flag_option=None, flag_index=None, username=None):
 
         print("Logged:", filenames[0])
 
+if opt.defaults is not None and os.path.isfile(opt.defaults):
+    with open(opt.defaults, "r", encoding="utf8") as f:
+        user_defaults = yaml.safe_load(f)

```

Comment:
What happens when the user specifies an incorrect file?

You might think it won't happen but there's always a chance of major pebkac so it could

Reply:
Pretty much the same thing that happens with the `--cli` arg currently. It just crashes. I'll add a fallback to defaults if the file can't be parsed at all but I don't know if it's worth doing more validation for this.